### DISCUSSION TITLE
Epiverse-TRACE developer space – Things that can go wrong when using renv

### DISCUSSION THREAD

**Discussion by giscus** [epiverse-trace/epiverse-trace.github.io#172] [Blog post discussion]
# Epiverse-TRACE developer space – Things that can go wrong when using renv

A place for Epiverse-TRACE developers to share their reflections, learnings, and showcase their work.

https://epiverse-trace.github.io/posts/renv-complications/

**Comment by b-rodrigues** [comment]
Hi Hugo, very interesting blog post! Thanks for sharing my blog posts on Nix :)
